Docs/user_guide/quantization_configuration.rst

```diff
@@ -39,53 +40,53 @@ When working with a new runtime with different rules, or for experimental purpos
 
         - is_symmetric:
-            An optional parameter. If included, possible settings include "True" and "False".
-            A "True" setting will place all activation quantizers in symmetric mode by default.
-            A "False" setting, or omitting the parameter altogether, will set all activation quantizers to asymmetric mode by default.
+            Optional. If included, value is "True" or "False".
+            "True" places all activation quantizers in symmetric mode by default.
+            "False", or omitting the parameter, sets all activation quantizers to asymmetric mode by default.
 
-    The "params" dictionary holds settings that will apply to all parameter quantizers in the model.
-    In this section, the following settings are available:
+    The "params" dictionary holds settings that apply to all parameter quantizers in the model.
+    The following settings are available:
 
         - is_quantized:
-            An optional parameter. If included, possible settings include "True" and "False".
-            A "True" setting will turn on all parameter quantizers by default.
-            A "False" setting, or omitting the parameter altogether, will disable all parameter quantizers by default.
+            Optional.  If included, value is "True" or "False".
+            "True" turns on all parameter quantizers by default.
+            "False", or omitting the parameter, disables all parameter quantizers by default.
 
         - is_symmetric:
-            An optional parameter. If included, possible settings include "True" and "False".
-            A "True" setting will place all parameter quantizers in symmetric mode by default.
-            A "False" setting, or omitting the parameter altogether, will set all parameter quantizers to asymmetric mode by default.
+            Optional.  If included, value is "True" or "False".
+            "True" places all parameter quantizers in symmetric mode by default.
+            "False", or omitting the parameter, sets all parameter quantizers to asymmetric mode by default.
 
-    Aside from the "ops" and "params" dictionary, additional settings governing quantizers in the model are available:
+    Outside the "ops" and "params" dictionaries, the following additional quantizer settings are available:
 
     - strict_symmetric:
-        An optional parameter. If included, possible settings include "True" and "False".
-        When set to "True", quantizers which are configured in symmetric mode will use strict symmetric quantization.
-        When set to "False" or omitting the parameter altogether, quantizers which are configured in symmetric mode will not use strict symmetric quantization.
+        Optional.  If included, value is "True" or "False".
+        "True" causes quantizers configured in symmetric mode to use strict symmetric quantization.
+       "False", or omitting the parameter, causes quantizers configured in symmetric mode to not use strict symmetric quantization.
```

Comment:
Change in indent here messes with the formatting

Reply:
Reformatted entire doc. Looks better now.